projects/kubernetes/Makefile

```diff
@@ -1,19 +1,10 @@
 all: build-container-images build-vm-images
 
-BOX_PLANS = kubernetes.rb mounts.rb
-
-build-container-images: $(BOX_PLANS)
-	for plan in $(BOX_PLANS) ; do \
-	  docker run --rm -ti \
-	    -v $(PWD):$(PWD) \
-	    -v /var/run/docker.sock:/var/run/docker.sock \
-	    -w $(PWD) \
-	      boxbuilder/box:master $$plan \
-	; done
-
-push-container-images: build-container-images cache-images
-	docker image push linuxkit/kubernetes:latest
-	docker image push linuxkit/kubernetes:latest-mounts
+build-container-images:
+	make -C kubernetes tag
+
+push-container-images: cache-images
+	make -C Kubernetes push
 	docker image push linuxkit/kubernetes:latest-image-cache-common
 	docker image push linuxkit/kubernetes:latest-image-cache-control-plane
```

Comment:
Was planning to do that when I reworked the build for those images.

You did make me notice `Kubernetes` in the line above though, should be lowercase...